site/gatsby-site/src/components/RelatedIncidents.js

```diff
@@ -33,6 +41,31 @@ const relatedReportsQuery = gql`
   }
 `;
 
+const reportsWithIncidentIds = async (reports, client) => {
```

Comment:
In the future, we should set up a schema relationship to avoid having to do this: https://github.com/responsible-ai-collaborative/aiid/blob/master/site/realm/data_sources/mongodb-atlas/aiidprod/incidents/relationships.json